modules/javafx.graphics/src/main/java/com/sun/glass/ui/headless/HeadlessView.java

```diff
@@ -0,0 +1,130 @@
+package com.sun.glass.ui.headless;
+
+import com.sun.glass.events.ViewEvent;
+import com.sun.glass.ui.Pixels;
+import com.sun.glass.ui.View;
+import java.util.Map;
+
+public class HeadlessView extends View {
+
+    private Map capabilities;
+    private int x = 0;
+    private int y = 0;
+    private long parentPtr = 0;
+    private Pixels pixels;
+
+    private boolean imeEnabled;
```

Comment:
The following fields are not read, but only assigned to in this class:
* `capabilities`
* `x`
* `y`
* `imeEnabled`

The field `parentPtr` is neither assigned nor read.

Reply:
removed capabilities and imeEnabled